t3/utils/flux.py

```diff
@@ -295,6 +295,15 @@ def set_batch_p(gas: ct.Solution,
     network.rtol = r_tol
     return network, reactor
 
+def closest_bigger_number(array, target):
+    # Filter the numbers greater than the target
+    candidates = [(num, i) for i, num in enumerate(array) if num > target]
+    # Find the closest bigger number
+    if candidates:
+        m, i = min(candidates, key=lambda x: x[0])
```

Comment:
Can you explain what is being returned as `m, i`? Looks to me like the `min()` func only returns one value, I'm probably missing something

Reply:
Let's use consistent variable names, for example if we call the content of the tuple pair `(num, i)`, then let's also call them `num, i` on this line if indeed they represent the same type of data

Reply:
Candidates is a list of tuples (num, i), with numbers from the original array which are bigger that the target number. Let's call this process as filtering out smaller numbers.
In the next step, the min function is used over a list of tuples, while the key for the min function is the first variable in tuple (which is the value number itself). The min function returns a value from the list (which is a tuple consists of a value and its index in the original array).